api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -42,11 +42,81 @@ This endpoint returns data corresponding to the latest data dictionary csv file.
 ### Data Dictonary [GET]
 
 + Response 200 (application/json)
-    + Attributes 
+    + Attributes
         + document (DataDictionary)
-    
+
+
+## Download Count [/api/v2/download/count/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Download Count Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (required, object)
+            + time_period (array[TimeRangeObject])
+
++ Response 200 (application/json)
+    + Attributes
+        + transaction_rows_gt_limit : true (required, boolean)
+            A boolean returning whether the transaction count is over the maximum row limit.
+
+## Reference Filter [/api/v1/references/filter/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Reference Filter Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (optional, object)
+
++ Response 200 (application/json)
+    + Attributes
+        + hash : `96982f90346b1360dc5fb0a97d4b23fa` (required, string)
+
+## Reference Hash [/api/v1/references/hash/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Reference Hash Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + hash : `96982f90346b1360dc5fb0a97d4b23fa` (required, string)
+
++ Response 200 (application/json)
+    + Attributes
+        + filter (optional, object)
```

Comment:
When you move to the Advanced Search file, you can use the existing [Filter Object](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/api_contracts/contracts/search/AdvancedSearch.md#filterobject-object) data structure.